pkgs/by-name/no/nostui/package.nix

```
@@ -0,0 +1,43 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+, darwin
+, stdenv
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "nostui";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "akiomik";
+    repo = pname;
```

this is considered an antipattern

```suggestion
    repo = "nostui";
```

---

Fixed in latest push, thanks for the feedback!